app/controllers/users_controller.rb

```diff
@@ -0,0 +1,14 @@
+class UsersController < ApplicationController
+    def show
+        @user = User.find(params[:id])
+    end
+    def update
```

Comment:
remove the update action, you dont use it

Reply:
done